### PR TITLE
config: customisable support references; Fix #243

### LINKED ISSUE #243: Removing reference to  https://its.cern.ch/jira/browse/RUCIO in rucio clis
Motivation
----------
We should replace https://its.cern.ch/jira/browse/RUCIO by the github issue link in the client code

2017-11-30 16:01:50,357 ERROR [Rucio exited with an unexpected/unknown error. Please rerun the last command with the '-v' option and submit a ticket with all the necessary information at https://its.cern.ch/jira/browse/RUCIO]

### PR DESCRIPTION
**Pull request by mlassnig**
config: customisable support references; Fix #243

### CODE REVIEW

File: bin/rucio [hunk 1]
```diff
@@ -229,20 +229,20 @@ def exception_handler(function):
...
                 logger.error(error)
                 logger.debug(traceback.format_exc())
-                logger.error("This means that the object doesn't have the property " + str(error) +
-                             ". This should never happen. Please rerun the last command with the '-v'" +
-                             " option and submit a ticket with all the necessary information at https://its.cern.ch/jira/browse/RUCIO")
-            return FAILURE
+                logger.error('The object is missing this property: ' + str(error))
+                logger.error('This should never happen. Please rerun the last command with the "-v" option to gather more information.')
+                logger.error('Please followup with all relevant information at: %s' % config_get('policy', 'support_email'))
+            raise
```
Comment (vingar) [inline bin/rucio]: just for info, it's recommended to split one string on several lines like that
```
s = ("this is a very"
      "long string too"
      "for sure ..."
     )
```
here:

```
logger.error('The object is missing this property: ' + str(error)
                  'logger.error('This should never happen. Please rerun the last command with the "-v" option to gather more information.')
```

**Comment by bari12** [comment]
Two things come to my mind:
- I think the variables should be called `support_url` and `rucio_url`. Could be both an email, or url, so I would just call it URL.
- "If it is a problem concerning Rucio itself" is maybe not strict enough. Maybe write: "If you are sure it is a porblem concerning Rucio itself" ... :)

**Comment by vingar** [comment]
Martin's comment makes me think that maybe  config_get('policy', 'support_email') can come from the server through the config api rather than the local configuration file.

**Comment by bari12** [comment]
But not being able to connect to the server might be an actual reason that somebody wants to contact the support 😆

**Comment by vingar** [comment]
:)

**Comment by mlassnig** [comment]
job is ok in travis, but github never gets the notification...

https://travis-ci.org/rucio/rucio/builds/342412518

**Review by bari12 — APPROVED** [review]

**Pull request merged**